pkg/scheduler/scheduler.go

```diff
@@ -96,6 +97,8 @@ type schedulerOptions struct {
 	profiles                   []schedulerapi.KubeSchedulerProfile
 	extenders                  []schedulerapi.Extender
 	frameworkCapturer          FrameworkCapturer
+	// Set the parallel thread count for parallism package
+	cputhreadcount int32
```

Comment:
let's use the same variable name `parallelism` for consistency

Reply:
I think we can leave the `CPUThreadCount` in the `parallelize` package for better understanding and readability, if we want this parallelism to have a  correlation with CPU Threads.